From c053d7b078b7ebc067c24fd43c728ec0dcd69bfd Mon Sep 17 00:00:00 2001
From: Clayton Craft <clayton@craftyguy.net>
Date: Sun, 5 Sep 2021 13:53:44 -0700
Subject: [PATCH] quirks: Pine64 PineBook Pro keyboard

The keyboard is 'internal' (built-in), and attached over usb.

Signed-off-by: Clayton Craft <clayton@craftyguy.net>
---
 quirks/50-system-pine64.quirks | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 quirks/50-system-pine64.quirks

diff --git a/quirks/50-system-pine64.quirks b/quirks/50-system-pine64.quirks
new file mode 100644
index 00000000..b7943270
--- /dev/null
+++ b/quirks/50-system-pine64.quirks
@@ -0,0 +1,6 @@
+[PineBook Pro Keyboard]
+MatchVendor=0x258A
+MatchProduct=0x001E
+MatchUdevType=keyboard
+MatchBus=usb
+AttrKeyboardIntegration=internal
-- 
2.33.0

